man/systemd-creds.xml

```diff
@@ -163,6 +163,20 @@
         and thus decryption is entirely automatic.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><command>has-tpm2</command></term>
+
+        <listitem><para>Reports whether the system is equipped with a TPM2 device usable for protecting
+        credentials. If the a TPM2 device has been discovered, is supported, and is being used by firmware,
+        by the OS kernel drivers and by userspace (i.e. systemd) this prints <literal>yes</literal> and exits
+        with exit status zero. If no such device is discovered/supported/used, prints
+        <literal>no</literal>. Otherwise prints <literal>partial</literal>. In either of these two cases
+        exits with non-zero exit status. It also shows three lines indicating separately whether drivers,
+        firmware and the system discovered/support/use TPM2.</para>
+
+        <para>Combine with <option>-q</option> to suppress the output.</para></listitem>
```

Comment:
Do we prefer long or short names when suggesting other cmdline options?

Reply:
actually, long ones.